gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/02_add_shared_policy_group.yml

```diff
@@ -15,6 +15,7 @@ databaseChangeLog:
               - column: {name: cross_id, type: nvarchar(64), constraints: { nullable: false } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.